stock_dashboard_api/views/stock_view.py

```diff
@@ -1,13 +1,15 @@
+from datetime import datetime
+
 from flask import Blueprint, request, make_response, jsonify, Response
-from flask import Blueprint, request, make_response, jsonify
 from flask.views import MethodView
 
 from stock_dashboard_api.models.stock_model import Stock
 
-
 MAX_STOCK_NAME_LENGTH = 16
 MAX_STOCK_COMPANY_NAME_LENGTH = 128
 
+DATETIME_PATTERN = '%y/%m/%d %H:%M:%S'
```

Comment:
This is another `DATETIME_PATTERN` and instead of `-` year, month and day is separated by `/`????